client/src/components/Header/index.js

```diff
@@ -9,12 +11,31 @@ import { Link } from '../helpers';
 
 import './header.css';
 
+import {
+  toggleDisplaySideNav,
+  toggleDisplayMenu
+} from '../layouts/components/guide/redux';
+
+const mapStateToProps = state => {
+  return {
+    displayMenu: state.guideNav.displayMenu
```

Comment:
The selector pattern seeks to decouple state from consumers.

Currently, if we changed the shape of the application state, we would have to visit these files so they still received the correct state. For example, if I changed the `guideNav` namespace.

If we used a selector, we could restructure the state and only have to worry about how the selector accessed the state required. The consumers would not know that the shape of state has changed.

**TLDR;** State consumers should not care for the shape of the state they are consuming, selectors to the rescue! 💪